ZaeXT/test-scripts/test_backend.py

```
@@ -0,0 +1,459 @@
+        for event in client.events():
+            if event.event == 'message':
+                full_response += event.data
+        
+        # d. 验证AI的回答是否包含了记忆中的关键信息
+        print(f"      AI对记忆问题的回复: {full_response.strip()}")
+        if "gopher king" in full_response.lower():
+            print_success("用户记忆功能验证成功！AI正确回答了基于记忆的问题。")
+        else:
+            print_fail("用户记忆功能验证失败，AI未能使用System Prompt中的记忆信息。")
+
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)